cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -42,10 +42,10 @@ spec:
           value: --no-supervisor -q
         resources:
           limits:
-            memory: 300Mi
+            memory: 300Mi # fluentd-gcp-memory-limit
```

Comment:
Yup, it's a comment, so that the file can still be used without running sed on it.